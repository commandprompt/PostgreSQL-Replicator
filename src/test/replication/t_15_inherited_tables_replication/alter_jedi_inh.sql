ALTER TABLE jedi INHERIT rebels;
