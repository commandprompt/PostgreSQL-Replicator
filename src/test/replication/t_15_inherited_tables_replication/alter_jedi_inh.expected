psql:.*:1: ERROR:  cannon inherit from replicated relation "rebels"
HINT:  enable replication for relation "jedi"
